src/interfaces/api-task-interface.ts

```diff
@@ -36,24 +36,25 @@ export interface ApiHeaderIF {
  * Note that only `business` and `header` are required.
  */
 export interface TaskTodoIF {
+  alteration?: AlterationIF
   annualReport?: any
   business: ApiBusinessIF
   changeOfAddress?: any
   changeOfDirectors?: any
+  changeOfRegistration?: any
+  consentContinuationOut?: any
+  conversion?: any
   correction?: any
+  courtOrder?: any
```

Comment:
Can we change it to CourtOrderIF instead of any?

Reply:
I have done this (see next commit) but it doesn't help us in any way. First of all, this data from the API (which we don't control). Secondly, we don't get or set any properties from this object. I think I'll undo this change.

However, the ApiFilingIF contains a data.order object that could use this type... Except the definition in Filings UI:
![image](https://user-images.githubusercontent.com/10002889/228632719-acac8866-8682-48ef-9f2b-c018a28d7ed0.png)
is better than the definition in the shared repo:
![image](https://user-images.githubusercontent.com/10002889/228632807-df642291-9a55-4497-be6f-1a996f74584e.png)

Yes, this is a bit of tech debt that would be nice to resolve, but I'm calling it out of scope for this ticket. Are you OK with that?

Reply:
Okay, got it. Thank you for the explanation! We can resolve that in another ticket by having a uniform interface. I'll be resolving this conversation.